holoviews/plotting/mpl/plot.py

```diff
@@ -19,12 +20,27 @@
 from .util import compute_ratios, fix_aspect
 
 
+@contextmanager
+def rc_context(rcparams):
```

Comment:
To avoid confusion maybe this should be ``_rc_context`` if we expect to use the decorator and not this context manager directly. Perhaps you could even inline this context manager inside ``mpl_rc_context``?